### DISCUSSION TITLE
Skipping some parts of dialogue

### DISCUSSION THREAD

**Discussion by AlmostHuman34** [SubtitleEdit/subtitleedit#8297] [General]
Heya, been using subtitleedit with purfview's faster-whisper with models large-v1 and large-v2. I edit long podcasts and I always have same issue. At some points there is always ",,," or empty line instead of a text. All files I edit are in Polish. Is there any way to fix blank text boxes? Or do I always have to fix it manually. I'm not that well experienced with some settings.  This issue has been present since times of at least 3.6 version of subtitleedit. Thank you in advance.

**Comment by niksedk** [comment]
Probably an issue with the whisper model... but you should post a sample audio file, so we/people can try it.

**Reply by AlmostHuman34** [reply]
Hmm I does happened randomly and I tried medium model as well, same results. I will search for certain audio part to link it here soon.
